api/provider/aws/service_create.go

```diff
@@ -1,8 +1,137 @@
 package aws
 
-import "github.com/quintilesims/layer0/common/models"
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
 
 func (s *ServiceProvider) Create(req models.CreateServiceRequest) (*models.Service, error) {
+	var (
```

Comment:
this block is redundant. Also it's a pattern for C, not Go

Reply:
Yeah, I was going back and forth on visibility of import vars vs not-redundancy. Didn't know it was a C pattern, so TIL.

✅